controller/transfer/record_transfer_manager.py

```diff
@@ -82,16 +82,12 @@ def import_records_and_rlas(
 
         if upload_task is not None:
             progress = ((idx + 1) / chunks_count) * 100
-            upload_task_manager.update_task(
-                project_id, upload_task.id, progress=progress
-            )
+            upload_task_manager.update_task(project_id, upload_task.id, progress=progress)
 
 
-def import_file(project_id: str, upload_task: UploadTask) -> None:
-    # load data from s3 and do transfer task/notification management
-    upload_task_manager.update_task(
-        project_id, upload_task.id, state=enums.UploadStates.PENDING.value
-    )
+def download_file(project_id: str, upload_task: UploadTask) -> str:
+    # TODO is copied from import_file and can be refactored because atm its duplicated code
```

Comment:
TODO

Reply:
Resolved the todo by refactoring duplicated code by using the method at all places